addons/product/views/product_views.xml

```diff
@@ -365,9 +365,10 @@
                 </xpath>
                 <field name="list_price" position="attributes">
                    <attribute name="attrs">{'readonly': [('product_variant_count', '&gt;', 1)]}</attribute>
+                   <attribute name="invisible">1</attribute>
```

Comment:
Doesn't `attrs` attribute expect domains ?
I think a separate invisible attribute is cleaner and clearer here :)

Reply:
> Doesn't `attrs` attribute expect domains ?

One could put a true leaf/domain (optimized to avoid queries etc.)
https://github.com/odoo/odoo/blob/561fee585643cdf3ea3af2ef548b2d5fd1d86a1c/odoo/osv/expression.py#L176

But I agree that the invisible node attribute is better